.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+// Place your settings in this file to overwrite default and user settings.
```

Comment:
thoughts on gitignoring this file? seems like local dev preferences - is it needed for local debug?